right/src/slave_scheduler.h

```diff
@@ -8,9 +8,8 @@
 
 // Macros:
 
-    #define SLAVE_COUNT (sizeof(Slaves) / sizeof(uhk_slave_t))
```

Comment:
@steamraven Why not keep the actual implementation of `SLAVE_COUNT`?

Reply:
I tried to just use that, but could not get it to compile.  It seams you can't specify the size of an array as reference to the size of the array. And if you leave the size blank, you get " invalid application of 'sizeof' to incomplete type 'uhk_slave_t[] "

Reply:
You're completely right. Thanks for the explanation!